workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Solutions.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Why Solutions.ts? you can use a name related to the enums here

Order.ts makes more sense